UEBPrep/app/views/cards/_form.html.erb

```diff
@@ -10,7 +10,31 @@
       </ul>
     </div>
   <% end %>
-
+    <% if @card.content_type_id == 1 %>
+        <h2>Text Content</h2>
+        <%= label_tag(:cardtext, "Text") %>
+        <%= text_area_tag( 'text', TextContent.find(@card.content_id).text , size: '8x8') %>
+    <% elsif @card.content_type_id == 2 %>
+        <h2>Card Title</h2>
+        <%= label_tag(:cardtitle, "Title") %>
+        <%= text_field_tag( 'title', TitledCardContent.find(@card.content_id).title) %>
+        <h2>Text Content</h2>
+        <%= label_tag(:cardtext, "Text") %>
+        <%= text_area_tag( 'text', TitledCardContent.find(@card.content_id).text , size: '8x8') %>
+    <% else %>
+        <h2>Card Title</h2>
+        <%= label_tag(:cardtitle, "Title") %>
+        <%= text_field_tag( 'title', ImageCardContent.find(@card.content_id).title) %>
+        <h2>Text Content</h2>
+        <%= label_tag(:cardtext, "Text") %>
+        <%= text_area_tag( 'text', ImageCardContent.find(@card.content_id).text , size: '8x8') %>
+        <h2>Alt Text Content</h2>
+        <%= label_tag(:cardalt, " Alt Text") %>
+        <%= text_area_tag( 'alt', ImageCardContent.find(@card.content_id).alt , size: '8x8') %>
+        <%= label_tag(:cardimagealt, "Image") %>
+        <%= link_to image_tag(@card.content.image.url(:med)), alt: @card.content.alt %>
+        <%= file_field_tag('uploadcardimage', accept: 'image/png,image/gif,image/jpeg')%>
+    <% end %>
```

Comment:
"could get" implying it's not already pretty messy. Not that Levi had a choice with the current Content organization.

As a first pass this looks fine to me.